org.knime.knip.base/src/org/knime/knip/base/nodes/util/tilelooper/TileIteratorLoopEndNodeFactory.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE knimeNode PUBLIC "-//UNIKN//DTD KNIME Node 2.0//EN" "http://www.knime.org/Node.dtd">
+<knimeNode icon="tileiteratorend.png" type="LoopEnd">
+	<name>Tile Loop End</name>
+
+	<shortDescription>
+		Collects tiles of Tile Loop Start and recombines them to images.
```

Comment:
I think something like: "Collects tiles **from** the Tile Loop Start **node** .." would be better.